test/play-pause-status.js

```diff
@@ -31,34 +31,20 @@ describe('A new session with Google Music', function () {
       expect(this.result).to.equal(2 /* PLAYING */);
     });
 
-    describe('and pause it', function () {
-      before(function pausePlayback (done) {
-        // Find and click the play/pause button
-        var browser = this.browser;
-        browser.elementByCssSelector('[data-id=play-pause]', function handleElement (err, el) {
-          // If there was an error, callback with it
-          if (err) {
-            return done(err);
-          }
-
-          // Otherwise, click our element
-          el.click(done);
-        });
-      });
-      browserMusicUtils.waitForPlaybackPause();
+    describe('and when called via API', function () {
```

Comment:
This sentence doesn't make much sense =/ Maybe say "and when retrieving playback state"?